tests/integration/server/users_api_test.py

```diff
@@ -222,6 +222,16 @@ def test_user_delete_api():
             .coerce_to("array")
             .run(conn)
         )
+
+        role_owners = (
```

Comment:
Looking at the earlier part of this test, the test user was never an owner to a role. You should create a test role using the test user first, assert that the user is actually a role owner, then call the Delete User API endpoint to make sure that your API endpoint actually removes the user from the role owners table.

Reply:
My apologies, I somehow missed the assertion on line 223 on my first look through the PR. Closing out this comment.